README.md

```diff
@@ -10,4 +10,7 @@ status](https://www.r-pkg.org/badges/version/admiralvaccine)](https://CRAN.R-pro
 [![Test
 Coverage](https://raw.githubusercontent.com/pharmaverse/admiralvaccine/badges/main/test-coverage.svg)](https://github.com/pharmaverse/admiralvaccine/actions/workflows/code-coverage.yml)
 
+
+### this is to be removed
```

Comment:
@federicobaratin @ap765541 @danielebottigliengo  can you all take a look at this and help me?